lib/hyku_addons/engine.rb

```diff
@@ -92,6 +94,20 @@ def reload_hyrax_config!
             config.geonames_username = Settings.geonames_username
           end
         end
+
+        def setup_tenant_cache
+          Rails.application.config.action_controller.perform_caching = Settings.cache_enabled && true
+          ActionController::Base.perform_caching = Rails.application.config.action_controller.perform_caching
+          Rails.application.config.cache_store = :redis_cache_store, { url: Redis.current.id }
+          Rails.cache = ActiveSupport::Cache.lookup_store(Rails.application.config.cache_store)
```

Comment:
Can these lines be skipped if `perform_caching == false`?

Reply:
good point